lib/p2p/Peer.ts

```diff
@@ -495,6 +499,22 @@ class Peer extends EventEmitter {
           this.handlePing(packet);
           break;
         }
+        case PacketType.DEAL_REQUEST: {
+          this.handleDealRequest(packet);
```

Comment:
Not really. Since it is async it can run async since we are not using anything from the response and are not dependent on it. 
Anyway, it is not async anymore

Reply:
OK, but for future reference not having an `await` statement for an async function allows for the possibility of an unhandled rejection error which would crash the process, so it's good to await even when we're not depending on the response. There's a lint rule for this actually which flagged the line.